pki/pki_test.go

```diff
@@ -0,0 +1,54 @@
+package pki_test
+
+import (
+	"testing"
+
+	"github.com/cloudflare/circl/pki"
+	"github.com/cloudflare/circl/sign/api"
+)
+
+func TestPEM(t *testing.T) {
+	for _, scheme := range api.AllSchemes() {
```

Comment:
This seems like an appropriate use for subtests.

```go
func TestPEM(t *testing.T) {
	for _, scheme := range api.AllSchemes() {
		t.Run(scheme.Name(), func(t *testing.T) {
			// ...
		}
	}
}
```

Reply:
Nice, didn't know about these.  Thanks!